src/Microsoft.VisualStudio.SlowCheetah.VS/Utilities/ProjectUtilities.cs

```diff
@@ -187,8 +188,11 @@ public static bool IsProjectWebApp(IVsProject project)
             if (project is IVsAggregatableProject aggregatableProject)
             {
                 aggregatableProject.GetAggregateProjectTypeGuids(out string projectTypeGuids);
-                List<string> guids = new List<string>(projectTypeGuids.Split(';'));
-                return guids.Contains(Guids.GuidWebApplicationString);
+                var guidRegex = new System.Text.RegularExpressions.Regex(@"[0-9A-F]{8}[-]?([0-9A-F]{4}[-]?){3}[0-9A-F]{12}", System.Text.RegularExpressions.RegexOptions.IgnoreCase);
+                return projectTypeGuids.Split(';')
+                                       .Where(x => guidRegex.Match(x).Success)
+                                       .Select(x => new Guid(guidRegex.Matches(x)[0].Value))
```

Comment:
I think you can just have this select and catch exceptions when attempting to create the new guid. This would remove invalid strings from consideration.